README.ru.md

```diff
@@ -1,42 +1,43 @@
 # sway
 
-sway - это [i3](https://i3wm.org/)-совместимый композитор [Wayland](http://wayland.freedesktop.org/).
-Больше информации в [FAQ](https://github.com/swaywm/sway/wiki). Присоединяйтесь к
-[IRC-каналу](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway на
+**[English][en]** - [日本語][ja] - [Français][fr] - [Українська][uk] - [Español][es] - [Polski][pl] - [中文-简体][zh-CN] - [Deutsch][de] - [Nederlands][nl] - [Русский][ru] - [中文-繁體][zh-TW] - [Português][pt] - [Danish][dk] - [한국어][ko] - [Română][ro]
```

Comment:
Other translated pages don't have the list of all languages. Since it becomes pretty easily outdated, can we omit it here as well?

Reply:
Sure, I made a commit